Jasmine/Common/Models/GameStatus.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+/// Denotes the status of the current game play.
+///
+/// - notStarted: the game has not started.
+/// - inProgress: the game is currently in progress.
+/// - endedWithWon: the game has ended and the user has won.
+/// - endedWithLost: the game has ended and the user has lost.
+/// - endedWithTimeOut: the game has ended and the user ran out of time.
+enum GameStatus {
+    case notStarted
+    case inProgress
+    case endedWithWon
+    case endedWithLost
```

Comment:
I don't think there's a need until we want to animate the difference in the way the user lost. Even so, `endedWithTimeOut` shouldn't be here.